velox/vector/arrow/Bridge.cpp

```diff
@@ -662,15 +838,14 @@ VectorPtr createStringFlatVector(
       std::move(stringViewBuffers),
       SimpleVectorStats<StringView>{},
       std::nullopt,
-      nullCount == -1 ? std::nullopt : std::optional<int64_t>(nullCount));
+      optionalNullCount(nullCount));
 }
 
 VectorPtr importFromArrowImpl(
-    const ArrowSchema& arrowSchema,
-    const ArrowArray& arrowArray,
+    ArrowSchema& arrowSchema,
+    ArrowArray& arrowArray,
```

Comment:
why do we need to drop the const here?

Reply:
We need to move the ownership when `isViewer` is false